### ISSUE TITLE
Help understanding LMM

### ISSUE THREAD

**Issue by DesmondSmith** [kbroman/lmmlite#3]
Thanks very much Karl for lmmlite and your corresponding blog entry, which were incredibly helpful in advancing my (meager) comprehension of LMMs in GWAS. However, I am ashamed to say I am still struggling with a full understanding some 6 years after your blog entry. If one were to code a GWAS LMM in lme4, would be model be roughly of the following form?

m1 <- lmer(y ~ s1 + s2 + s3... + (1|s1) + (1|s2) + (1|s3)..., data = pheno_snps, REML = TRUE)

In other words, are snps s1, s2, s3... essentially treated as both fixed effects (snp effect) and random intercepts (correctinng for population structure)? (But done much more cleverly in lmmlite with rotation of the phenotypes and covariates using the eigendecomposition of K).

Many thanks again for your help!

Des

**Comment by kbroman** [comment]
The way we handle the polygenic effect in lmmlite does not fit within the model formula scheme used in lme4. There's a related package [pedigreemm](https://cran.r-project.org/package=pedigreemm) for this, but I've not used it.

**Comment by DesmondSmith** [comment]
Thanks very much, Karl! -- Des
